server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/environment/EnvironmentService.java

```diff
@@ -84,12 +88,31 @@ public class EnvironmentService {
 
     private final List<Consumer<EnvironmentDeleteEvent>> preDeleteHooks = new ArrayList<>();
     private final List<Consumer<EnvironmentDisableEvent>> preDisableHooks = new ArrayList<>();
+    private final Set<String> DEFAULT_ENV_NAMES = new HashSet<>();
+
+    @PostConstruct
+    public void init() {
+        DEFAULT_ENV_NAMES.add("开发");
+        DEFAULT_ENV_NAMES.add("测试");
+        DEFAULT_ENV_NAMES.add("生产");
+        DEFAULT_ENV_NAMES.add("默认");
+        DEFAULT_ENV_NAMES.add("開發");
+        DEFAULT_ENV_NAMES.add("測試");
+        DEFAULT_ENV_NAMES.add("生產");
+        DEFAULT_ENV_NAMES.add("默認");
+        DEFAULT_ENV_NAMES.add("dev");
+        DEFAULT_ENV_NAMES.add("sit");
+        DEFAULT_ENV_NAMES.add("prod");
+        DEFAULT_ENV_NAMES.add("default");
```

Comment:
should we add other reserved naming, such as pre-prod， 预发

Reply:
I assume not, as the reason why we restrict names here is that dev/sit/prod/default is built-in environments, while pre/预发 are not. Users could be able to define pre/预发 themselves and we should not restrict this action, otherwise allowing users to define customed environments is meaningless.